aips/aip-43.md

```diff
@@ -39,6 +39,20 @@ AIP 36 removed one serialization point. We can remove the rest with:
 - using Aggregators for total_supply and current_supply counters in the collection.move
 - parallelize event creation, using module events that remove sequence number altogether 
 
+In addition, we will add new API to mint tokens:
+```
+  public fun create_with_index_in_name(
```

Comment:
Is create_numbered_token a better name? cc @gregnazario the naming master

Reply:
open to any name :) will wait for @gregnazario 's thoughts, and update to create_numbered_token or something better